python/adelphi/setup.py

```diff
@@ -13,17 +13,39 @@
 if not PY3:
     dependencies.append('backports.tempfile ~= 1.0')
 
+long_description = ""
+with open("README.md") as f:
+    long_description = f.read()
+
 setup(
-   name='adelphi',
-   version='0.1.0',
-   author='Gianluca Righetto',
-   author_email='gianluca.righetto@datastax.com',
+    name='adelphi',
+    url='https://github.com/datastax/adelphi',
+    description='Tooling for Adelphi',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    version='0.1.0',
+    project_urls={
+        'Source': 'https://github.com/datastax/adelphi',
+        'Issues': 'https://github.com/datastax/adelphi/issues',
+    },
```

Comment:
python-driver includes a "Documentation" link here which presumably shows up in the "Project links/Documentation" section on PyPI.  This documentation would presumably be specific to the Python package so we could make this a link to the README on Github... not sure what else would work there.

Reply:
If we can link back to our docs in the readme that seems like the best scenario so that we only have to manage it in one place over time.

Reply:
I wondered about that @jdonenine, but I held off since we were already linking to lots of stuff on Github.  cassandra-driver links to some DataStax-managed docs but I think that might just be a historical accident; I'm not sure that's the model we should aim for going forward.

I'll make this a link to the README for now, although weirdly it'll be a link to the README being added by this very pull request.  Remember that the docs we link to here should be specific to the Python package in question so we don't want to link to https://github.com/datastax/adelphi/blob/master/README.md.... I think we'd want https://github.com/datastax/adelphi/blob/master/python/adelphi/README.md instead.

Reply:
👍  on it needing to be the python script specific README, not the general repo one.